src/components/cluster.js

```diff
@@ -1,6 +1,10 @@
-import React from 'react';
+import React, {useEffect} from 'react';
 
 function Cluster(props) {
+  useEffect(() => {
+    document.title = "Coronavirus spread in India | Visual Map"
+ }, []);
```

Comment:
@Rajesh-Narayanappa87 very minor thing. You might want to fix indent over here. 
Cheers!

Reply:
Fixed it..

Reply:
Mind doing it here as well? https://github.com/covid19india/covid19india-react/pull/329/files#diff-08973bc23d3dd7488b8d243c028373bfR6
Thanks

Reply:
done